sdk/identity/identity/src/credentials/accessToken.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export interface AccessToken {
```

Comment:
@schaabs Updating `getToken` to return `AccessToken` instead of `string` would be in a separate PR, correct?

Reply:
If we're sure we want to make that change, I can do that today before merging!

Reply:
Log an issue for this which we can use to take it to a conclusion and make changes outside of this PR

Reply:
Logged issue https://github.com/Azure/azure-sdk-for-js/issues/3529 to track this work.